aim/db/migration/alembic_migrations/versions/5e29ae4f45e6_host_domain_add_primary_key.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2017 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Migrate HostDomainMapping to HostDomainMappingV2
+Revision ID: 5e29ae4f45e6
+Revises: 593d228d2fb4
+Create Date: 2017-10-25 16:39:33.713695
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5e29ae4f45e6'
+down_revision = '593d228d2fb4'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql.expression import delete
+
+AIM_HOST_DOMAIN_MAPPING = 'aim_host_domain_mapping'
+AIM_HOST_DOMAIN_MAPPING_V2 = 'aim_host_domain_mapping_v2'
+
+# A model of the old domains table
+domains = sa.Table(AIM_HOST_DOMAIN_MAPPING, sa.MetaData(),
```

Comment:
do we still need this since the old table still exists in this approach?

Reply:
I got that but why do u have to spell out all the details of this table? I will think the table name aim_host_domain_mapping itself should suffice, right?

Reply:
You need to declare the columns you want to access -- if you just declare the table name, it will assume it has no columns.

Reply:
the point here is that this is the existing table and in this approach we are not modifying this table so we should just import the model class then re-use that instead of creating the same model again locally.